src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -236,6 +262,8 @@ def attach_method(self, resource_id):
 
     def setup_lambda_api(self):
         """A wrapper for all the steps needed to setup the integration."""
+        self.create_resource(self.parent_id)
+        self.attach_method(self.resource_id)
```

Comment:
I like having the `attach_method()` call here in case someone wants to override this Method and it's clearer to read. Passing a class attribute as an argument seems a little strange, but no harm.